src/__init__.py

```diff
@@ -9,6 +9,14 @@
     os.environ["PATH"] += os.pathsep + extra_dll_dir
     os.add_dll_directory(extra_dll_dir)
 
+# TODO: This is an ugly workaround to add dll for fortran compilers, if this is in user space
+# The fortran compiler can be different from gfortran, or can be different from what was used when py3nj is installed 
+if sys.platform == "win32":
+    import subprocess
+    result = subprocess.run(['where', 'gfortran'], stdout=subprocess.PIPE)
+    path = os.path.dirname(result.stdout.decode('utf-8'))
+    os.add_dll_directory(path)
+
```

Comment:
As my local windows has DLLs in the same directory where `gfortran` is located, I added this ugly workaround. This works in my environment, and I hoped it works for many systems, but it does not.
We definitely need a better solution.